update/Pos/Update/Core/Types.hs

```diff
@@ -151,6 +154,9 @@ mkSystemTag tag | T.length tag > systemTagMaxLength
                 | otherwise
                     = pure $ SystemTag tag
 
+currentSystemTag :: SystemTag
+currentSystemTag = $(runQ [| SystemTag (T.pack (os ++ "_" ++ arch)) |])
```

Comment:
Very good. But let's have backwards-compatibility. I.e. systemTag `win64` should still remain `win64`. Not `minigw_x86_64`

BTW, can we log current system tag in compile-time?

Reply:
@georgeee Only got around to this now. What do you mean compile-time, just a `putStrLn` or using `log-warper` here (which I don't think is feasible)?

Reply:
`putStrLn` should be fine. Is it doable?